fix: Replace all occurrences of open-source-programs-backend

HeroicHitesh opened the pull request:
### Description

Replace all occurrences of `open-source-programs-backend` with `anitab-forms-backend`

Fixes #120 

### Type of Change:
<!--**Delete irrelevant options.**-->

- Documentation

### Checklist:
<!--**Delete irrelevant options.**-->

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

Commit:
fix: Replace all occurrences of open-source-programs-backend (#120)

codesankalp reviewed (dismissed):
@HeroicHitesh Good Work :+1:.
I hope there are no `open-source-programs-backend` after this.
But thanks for your contribution.

devkapilbansal reviewed (dismissed):
LGTM :+1:

isabelcosta commented on README.md:
```diff
@@ -184,7 +184,7 @@ Documentation for the project is hosted [here](https://osp-backend-docs.surge.sh
 - [GSoC 2020 Project Meeting Minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit) [Old].
 
-For more information, you can read [backend project wiki](https://github.com/anitab-org/open-source-programs-backend/wiki) and the [web project wiki](https://github.com/anitab-org/open-source-programs-web/wiki).
+For more information, you can read [backend project wiki](https://github.com/anitab-org/anitab-forms-backend/wiki) and the [web project wiki](https://github.com/anitab-org/open-source-programs-web/wiki).
```
```suggestion
For more information, you can read [backend project wiki](https://github.com/anitab-org/anitab-forms-backend/wiki) and the [web project wiki](https://github.com/anitab-org/anitab-forms-web/wiki).
```

isabelcosta commented on osp-backend-docusaurus/docs/2020-Bismita-Guha.md:
```diff
@@ -103,7 +103,7 @@ I am a sophomore in IIT Roorkee, doing my majors in Metallurgical Engineering. I
 ##
 ### Weekly Status Report for Week 4
-- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#32](https://github.com/anitab-org/open-source-programs-web/pull/32), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35), [#27](https://github.com/anitab-org/open-source-programs-backend/pull/27) & [#25](https://github.com/anitab-org/open-source-programs-backend/pull/25)<br ...
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#32](https://github.com/anitab-org/open-source-programs-web/pull/32), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35), [#27](https://github.com/anitab-org/anitab-forms-backend/pull/27) & [#25](https://github.com/anitab-org/anitab-forms-backend/pull/25)<br/>2. Worked on r ...
```
```suggestion
- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#32](https://github.com/anitab-org/anitab-forms-web/pull/32), [#35](https://github.com/anitab-org/anitab-forms-web/pull/35), [#27](https://github.com/anitab-org/anitab-forms-backend/pull/27) & [#25](https://github.com/anitab-org/anitab-forms-backend/pull/25)<br/>2. Worked on reviews by mentors on Previous PRs<br/>3. Attended Team Meeting for OSP<br/>4. Attended Work Session with Abha and Siddharth<br/>5. Attended 1:1 with Abha, Siddharth & Monal<br/>6. Worked on Weekly Blog<br/><br/>
```

isabelcosta commented on osp-backend-docusaurus/docs/2020-Bismita-Guha.md:
```diff
@@ -114,7 +114,7 @@ I am a sophomore in IIT Roorkee, doing my majors in Metallurgical Engineering. I
 ## Second Coding Phase (June 30, 2020 - July 27, 2020)
 ### Weekly Status Report for Week 5
-- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/open-source-programs-backend/pull/28), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35) & [#36](https://github.com/anitab-org/open-source-programs-web/pull/36)<br/>2. Work ongoing on issue [#19](https://github.com/anitab-org/open-source- ...
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/anitab-forms-backend/pull/28), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35) & [#36](https://github.com/anitab-org/open-source-programs-web/pull/36)<br/>2. Work ongoing on issue [#19](https://github.com/anitab-org/open-source-programs ...
```
```suggestion
- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/anitab-forms-backend/pull/28), [#35](https://github.com/anitab-org/anitab-forms-web/pull/35) & [#36](https://github.com/anitab-org/anitab-forms-web/pull/36)<br/>2. Work ongoing on issue [#19](https://github.com/anitab-org/anitab-forms-web/issues/19)<br/>3. Attended 1:1 with May, Monal, Abha & Siddharth<br/>4. Attended Work Session with Mentors<br/>Attended Team Meeting for OSP<br/>5. Speed up the work on the Documentation<br/>6. Attended extra meeting with Monal and Abha<br/>7. Fill in First evaluation<br/><br/>
```

isabelcosta commented on osp-backend-docusaurus/docs/2020-Bismita-Guha.md:
```diff
@@ -124,7 +124,7 @@ I am a sophomore in IIT Roorkee, doing my majors in Metallurgical Engineering. I
 ##
 ### Weekly Status Report for Week 6
-- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/open-source-programs-backend/pull/28), [#39](https://github.com/anitab-org/open-source-programs-web/pull/39) & [#40](https://github.com/anitab-org/open-source-programs-web/pull/40)<br/>2. Worked on issues [#12](https://github.com/anitab-org/open-source-progr ...
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/anitab-forms-backend/pull/28), [#39](https://github.com/anitab-org/open-source-programs-web/pull/39) & [#40](https://github.com/anitab-org/open-source-programs-web/pull/40)<br/>2. Worked on issues [#12](https://github.com/anitab-org/open-source-programs-web/ ...
```
```suggestion
- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/anitab-forms-backend/pull/28), [#39](https://github.com/anitab-org/anitab-forms-web/pull/39) & [#40](https://github.com/anitab-org/anitab-forms-web/pull/40)<br/>2. Worked on issues [#12](https://github.com/anitab-org/anitab-forms-web/issues/12), [#13](https://github.com/anitab-org/anitab-forms-web/issues/13) & [#8](https://github.com/anitab-org/anitab-forms-backend/issues/8)<br/>3. Attended Team Meeting and Work Session<br/>4. Attended 1:1 with May, Monal, Abha & Siddharth<br/>5. Worked on Documentation<br/>6. Created issues for the community<br/><br/>
```

isabelcosta commented on osp-backend-docusaurus/docs/2020-Bismita-Guha.md:
```diff
@@ -144,7 +144,7 @@ I am a sophomore in IIT Roorkee, doing my majors in Metallurgical Engineering. I
 ##
 ### Weekly Status Report for Week 8
-- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#39](https://github.com/anitab-org/open-source-programs-web/pull/39), [#40](https://github.com/anitab-org/open-source-programs-web/pull/40) & [#29](https://github.com/anitab-org/open-source-programs-backend/pull/29)<br/>2. Attended Team Meeting and Work Session<br/>3. Attended 1:1 with May, M ...
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#39](https://github.com/anitab-org/open-source-programs-web/pull/39), [#40](https://github.com/anitab-org/open-source-programs-web/pull/40) & [#29](https://github.com/anitab-org/anitab-forms-backend/pull/29)<br/>2. Attended Team Meeting and Work Session<br/>3. Attended 1:1 with May, Monal, Ab ...
```
```suggestion
- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#39](https://github.com/anitab-org/anitab-forms-web/pull/39), [#40](https://github.com/anitab-org/anitab-forms-web/pull/40) & [#29](https://github.com/anitab-org/anitab-forms-backend/pull/29)<br/>2. Attended Team Meeting and Work Session<br/>3. Attended 1:1 with May, Monal, Abha & Siddharth<br/>4. Worked on Documentation<br/>5. Created issues for the community<br/><br/>
```

isabelcosta commented on osp-backend-docusaurus/docs/Fork,-Clone,-Remote-and-Pull-Request.md:
```diff
@@ -3,43 +3,43 @@ id: Fork,-Clone,-Remote-and-Pull-Request
 
 ### Working on an Issue
-Now that you have setup the project, you need to search for anything to work on which interests you. Have a look at the [Backend Issues](https://github.com/anitab-org/open-source-programs-backend/issues) or [Frontend Issues](https://github.com/anitab-org/open-source-programs-web/issues), whichever interests you. 
+Now that you have setup the project, you need to search for anything to work on which interests you. Have a look at the [Backend Issues](https://github.com/anitab-org/anitab-forms-backend/issues) or [Frontend Issues](https://github.com/anitab-org/open-source-programs-web/issues), whichever interests you. 
```
```suggestion
Now that you have setup the project, you need to search for anything to work on which interests you. Have a look at the [Backend Issues](https://github.com/anitab-org/anitab-forms-backend/issues) or [Frontend Issues](https://github.com/anitab-org/anitab-forms-web/issues), whichever interests you. 
```

isabelcosta commented on osp-backend-docusaurus/docs/2020-Bismita-Guha.md:
```diff
@@ -155,7 +155,7 @@ I am a sophomore in IIT Roorkee, doing my majors in Metallurgical Engineering. I
 ## Third Coding Phase (July 28, 2020 - August 24, 2020)
 ### Weekly Status Report for Week 9
-- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#41](https://github.com/anitab-org/open-source-programs-web/pull/41) & [#31](https://github.com/anitab-org/open-source-programs-backend/pull/31)<br/>2. Worked on Issues[#11](https://github.com/anitab-org/open-source-programs-backend/issues/11), [#12](https://github.com/anitab-org/open-source- ...
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#41](https://github.com/anitab-org/open-source-programs-web/pull/41) & [#31](https://github.com/anitab-org/anitab-forms-backend/pull/31)<br/>2. Worked on Issues[#11](https://github.com/anitab-org/anitab-forms-backend/issues/11), [#12](https://github.com/anitab-org/anitab-forms-backend/issues/ ...
```
```suggestion
- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#41](https://github.com/anitab-org/anitab-forms-web/pull/41) & [#31](https://github.com/anitab-org/anitab-forms-backend/pull/31)<br/>2. Worked on Issues[#11](https://github.com/anitab-org/anitab-forms-backend/issues/11), [#12](https://github.com/anitab-org/anitab-forms-backend/issues/12), [#8](https://github.com/anitab-org/anitab-forms-web/issues/8), [](https://github.com/anitab-org/anitab-forms-web/issues/9) & [#10](https://github.com/anitab-org/anitab-forms-web/issues/10)<br/>3. Attended Team Meeting and Work Session<br/>4. Attended 1:1 with May, Monal, Abha & Siddharth<br/>5. Worked on Documentation<br/>6. Created issues for the community<br/><br/>
```

isabelcosta reviewed (changes requested):
Thank you for contributing @HeroicHitesh !

I noticed, some references to open source programs still, I made suggestions, feel free to apply them. I think I got all occurrences. I also saw some occurrences from the zulip channel, but that could be done on another issue.

Could you please make the changes and I can then approve?

Commit:
fix: Replace all occurrences of open-source-programs

HeroicHitesh commented:
> I noticed, some references to open source programs still

@isabelcosta Actually, the issue mentioned removing occurrences of `open-source-programs-backend` so earlier I replaced them only
> Could you please make the changes and I can then approve?

Now, I have removed occurrences of `open-source-programs` too. Let me know if anything still doesn't look right.

isabelcosta commented:
@HeroicHitesh so sorry 😅 I did not notice that little requirement. Will approve :)

isabelcosta reviewed (approved):
Amazing work @HeroicHitesh !
Thank you for your contribution!

codesankalp reviewed (approved):
LGTM :tada:
